src/github-web-worker.js

```diff
@@ -167,6 +167,11 @@ const api = {
         content: JSON.stringify(value)
       };
 
+      if (currentFiles && currentFiles.hasOwnProperty(this.oldBackupFileName)) {
+        // found old backup file, it will be deleted
+        files[this.oldBackupFileName] = null;

```

Comment:
By old backup file I mean the file from the previous (or rather current) backup system, that is _project-viewer.json_. There is no point in renaming it as we would just overwrite it during this very backup anyway.